en/sensor/sfxx_lidar.md

```diff
@@ -0,0 +1,43 @@
+# LightWare SF1XX Lidar
+
+LightWare develops a range of light-weight, general purpose, laser altimeters ("Lidar") suitable for mounting on UAVs. These are useful for applications including terrain following, precision hovering (e.g. for photography), warning of regulatory height limits, anti-collision sensing etc.
+
+![LightWare SF11/C Lidar](../../assets/hardware/sensors/sf11c_120_m.jpg)
+
+PX4 supports the following LightWare Lidar rangefinders through the I2C bus (only):
+* [SF10/A](http://lightware.co.za/shop2017/drone-altimeters/26-sf10a-25-m.html) (25 m)
+* [SF10/B](http://lightware.co.za/shop2017/drone-altimeters/25-sf10b-50-m.html) (50 m)
+* SF10/C (100m) (Discontinued)
+* [SF11/C](http://lightware.co.za/shop2017/drone-altimeters/44-sf11c-120-m.html) (120 m)
+* [LW20](http://lightware.co.za/shop2017/drone-altimeters/51-lw20-100-m.html) (100 m) - Waterproofed (IP67) with servo for sense-and-avoid applications
+
+
+## Configuring Lidar
+
+Newer sensors/versions require no additional configuration.
+
+> **Tip** Older sensors may be miss-configured to have an I2C address equal to `0x55` (this conflicts with `rgbled` module).
+> You can check the address by connecting the sensor via USB (it has an internal USB to serial converter), run terminal, press **space**.
+> If the I2C address is equal to `0x66` the sensor can be used with PX4.
```

Comment:
1. I assume this only works on Linux? What do you do on Windows?
2. What do you do if it has the  wrong address - are Lidar user-writable?

Reply:
@ChristophTobler "Apparentl" you can check the I2C address by connecting a sensor to your computer via USB and running terminal. Pressing space should display the address.
That sounds quite odd
- It does mean connect to a computer right?
- Is this just Ubuntu?
If you don't know, is there anyone else you would recommend we ask?

Reply:
Never heard of this, sorry... Where did you get this information from? I haven't seen it on the Devguide. Just check who wrote it, I guess.

Reply:
There is no information. So I will replace this with a "hint" to check out http://manpages.ubuntu.com/manpages/zesty/man8/i2cdetect.8.html